exercises/practice/binary-search/binary-search.spec.coffee

```diff
@@ -37,13 +37,13 @@ describe 'Binary Search', ->
       new BinarySearch(array).find 7
     .toThrow new Error 'value not in array'
 
-  xit "a value smaller than the array's smallest value is not found", ->
+  xit 'a value smaller than the array's smallest value is not found', ->
```

Comment:
```suggestion
  xit 'a value smaller than the array\'s smallest value is not found', ->
```

Reply:
I should also have read the string's content... -.-